Sources/Examples/EchoNIO/Generated/echo.grpc.swift

```diff
@@ -0,0 +1,73 @@
+//
```

Comment:
Could you use a symlink to the existing variant of this file (somewhere in `Tests/SwiftGRPCNIOTests`)? Or convert that file into a symlink pointing to this one (depending on how we've been handling this for the non-NIO case)?

Reply:
Done